activities/twenty_three_and_me/models.py

```diff
@@ -28,6 +18,9 @@ class Meta:
     user = fields.AutoOneToOneField(settings.AUTH_USER_MODEL,
                                     related_name='twenty_three_and_me')
 
+    # As of Feb 2016, we believe the the upload_to defined below is always
+    # overridden by the DropzoneS3UploadFormView upload_to definition.
+    # Maintaining here in case we use the version defined below in the future.
     genome_file = models.FileField(upload_to=get_upload_path, max_length=1024,
```

Comment:
anything we're only keeping to use in the future I would just remove because it will always be in git history :)